CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 Currently, this repo is in Prerelease. When it is released, this project will adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 ========
 
+## 0.9.1
+
+### Changes
+
+-   `Breadcrumbs`'s background color now reflects the app's globally applied section modifier, such as `.nypl--locations`. It recognizes these for `--locations`, `--whats-on`, `--research`, and `--books-and-more`.
```

Comment:
Does this need to be changed to reflect https://github.com/NYPL/nypl-design-system/issues/301#issuecomment-666517996

Reply:
That is what this comment is saying, but your question illustrates that that's not clear. How would you word this?

Reply:
OHHH ok so what I understand (after looking at your comment below) is that you now will have 
```
<body className="books-and-more">
<DS.breadcrumb /> 
</body> 
```

And these styles will be applied.  

Without the comment below, it seemed like you needed to pass in 
`<DS.Breadcrumb className="books-and-more" />` which is not what we want.  

I think updates to the Readme on what the section globals are and how to use them might be useful here.